inst/include/child_model_simulation.hpp

```diff
@@ -169,6 +169,26 @@ void calc_hiv_negative_pop(int time_step,
   }//end s
 }
 
+template<typename ModelVariant, typename real_type>
+void adjust_hiv_births(int time_step,
+                           const Parameters<ModelVariant, real_type> &pars,
+                           const State<ModelVariant, real_type> &state_curr,
+                           State<ModelVariant, real_type> &state_next,
+                           IntermediateData<ModelVariant, real_type> &intermediate) {
+  const auto demog = pars.base.demography;
+  const auto cpars = pars.children.children;
+  static_assert(ModelVariant::run_child_model,
+                "adjust_hiv_births can only be called for model variants where run_child_model is true");
+  constexpr auto ss = StateSpace<ModelVariant>().base;
+
+if(cpars.abortion(time_step,1) == 1){
+  state_next.children.hiv_births -= state_next.children.hiv_births * cpars.abortion(time_step,0);
```

Comment:
if you wanna be slick with it (completely optional change for fun)
```suggestion
  state_next.children.hiv_births *= 1 - cpars.abortion(time_step,0);
```

Reply:
To be honest I will forget what this does in the future.